lib/db.ts

```diff
@@ -17,12 +19,23 @@ export interface Note {
   dateAdded: Date;
   dateUpdated: Date;
   // position: number;
+  userId?: string; // the id of the user that created the note
+}
+
+export interface User {
+  id: string;
+  email: string;
+  password: string;
+  settings: {
+    theme: string;
+  };
```

Comment:
I think it will be easier to reason about different settings, and the User object as a whole, if all of them are grouped under a common parent